pod/perlreapi.pod

```diff
@@ -791,21 +859,24 @@ C<< ->offs[0].start/end >> represents C<$&> (or
 C<${^MATCH}> under C</p>) and C<< ->offs[paren].end >> matches C<$$paren> where
 C<$paren >= 1>.
 
-=head2 C<precomp> C<prelen>
+=head2 C<RX_PRECOMP> C<RX_PRELEN>
 
-Used for optimisations.  C<precomp> holds a copy of the pattern that
-was compiled and C<prelen> its length.  When a new pattern is to be
+Used for optimisations.  C<RX_PRECOMP> holds a copy of the pattern that
+was compiled and C<RX_PRELEN> its length.  When a new pattern is to be
 compiled (such as inside a loop) the internal C<regcomp> operator
-checks if the last compiled C<REGEXP>'s C<precomp> and C<prelen>
+checks if the last compiled C<REGEXP>'s C<RX_PRECOMP> and C<RX_PRELEN>
 are equivalent to the new one, and if so uses the old pattern instead
 of compiling a new one.
 
 The relevant snippet from C<Perl_pp_regcomp>:
 
-	if (!re || !re->precomp || re->prelen != (I32)len ||
-	    memNE(re->precomp, t, len))
+	if (!re || !RX_PRECOMP(re) || RX_PRELEN(re) != (I32)len ||
+	    memNE(RX_PRECOMP(re), t, len))
```

Comment:
I don't see that code in pp_regcomp.

I suspect this logic in re_op_compile replaces it:
```
    if (   old_re
        && !recompile
        && cBOOL(RX_UTF8(old_re)) == cBOOL(RExC_utf8)
        && ( RX_COMPFLAGS(old_re) == ( orig_rx_flags & RXf_PMf_FLAGCOPYMASK ) )
        && RX_PRELEN(old_re) == plen
        && memEQ(RX_PRECOMP(old_re), exp, plen)
        && !runtime_code /* with runtime code, always recompile */ )
```

Reply:
I guess I'll just remove the expression, not sure if it's really that helpful.